### ISSUE TITLE
verifyPhoneNumber () throws ERROR Error: Uncaught (in promise): Invalid phone number

### ISSUE THREAD

**Issue by shantanum91** [arnesson/cordova-plugin-firebase#516]
Code:
```
        let phoneNumberString = "+91" + this.profile.phone;
        console.log(phoneNumberString); //+91xxxxxxxxxx
        this.firebase.verifyPhoneNumber(phoneNumberString,60).then(data=> {
            console.dir(data);
        });
```

Throws:
ERROR Error: Uncaught (in promise): Invalid phone number

**Comment by Vivek-abstract** [comment]
Even I am getting the same error. Did you manage to solve it?

**Comment by sadjous** [comment]
Please nead some help about this error, it work for some GSM numbers and doesn't for others
i don't know wheter my phone number is not supported by firebase
thanks

**Comment by sharifullahsh** [comment]
I also have error Invalid phone number have you manage it

**Comment by Vivek-abstract** [comment]
No I didn't solve it but I switched to Facebook Account kit for phone verification. Rest everything is in Firebase.

**Comment by soumak77** [comment]
should be fixed by version 1.0.2
